lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -141,4 +149,39 @@ sub press_ok_clone {
     send_key($self->{ok_clone_shortcut});
 }
 
+sub modify_create_new_partition_table {
+    my ($self, $item) = @_;
+    assert_screen(EXPERT_PARTITIONER_PAGE);
+    _select_system_view_section();
+    send_key_until_needlematch(SELECTED_HARD_DISKS, 'down');
+    send_key($self->{modify_hard_disk});
+    send_key_until_needlematch(SELECTED_CREATE_NEW_TABLE, 'down');
+    send_key "ret";
+}
+
+sub select_msdos_partition_table {
+    my ($self) = @_;
+    assert_screen(NEW_PARTITION_TABLE_TYPE);
+    send_key($self->{select_msdos_shortcut});
+    send_key "ret";
+}
+
+sub confirm_deleting_current_devices {
+    my ($self) = @_;
+    assert_screen(DELETING_CURRENT_DEVICES);
+    send_key($self->{press_yes});
+}
+
+sub switch_to_partitions_tab {
```

Comment:
There is already select_partitions_tab, you should modify the existing one instead.

Reply:
so you use better OOP approach for pressing the key, but then `select_partitions_tab` could be completely replaced, right? so we don't have duplicated code.

Reply:
Ok, but there is "send_key('alt-p')" instead of using "send_key($self->{partitions_tab_shortcut}" . Should I change that line in the 'select_partitions_tab' ?

Reply:
done